### PR TITLE
Add internal review dashboard

### PR DESCRIPTION
**Pull request by LordExodius**
closes/addresses #212 

# Quick Overview of Changes

- add `/internal/dashboard` page for reviews
- add table showing all previous and current reviews you've done
- add edit button linked to `/internal/review?applicant=<id>`

image

# Checklist

- [x] If any frontend changes were made, include a screenshot/demo in the overview
- [x] Checked all uses of changed component(s)/function(s)
- [ ] Check that CI is passing.
- [ ] If no (re-)reviews after 1-2 days, ping the web leads on the discord to remind them to review the PR.

@basokant @ArsalaanAli

### CODE REVIEW

File: src/components/data-table.tsx
Comment (basokant) [inline src/components/data-table.tsx:1]: can u move this to components/ui/

File: src/components/columns.tsx
Comment (basokant) [inline src/components/columns.tsx:1]: rename this components/internal/review-columns.tsx or smth

File: src/components/columns.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        return <Link href={`./review?applicant=${userId}`}>✏️</Link>;
+      },
+    },
+    {
+      accessorKey: "applicant",
+      header: "Name",
+      cell: ({ row }) => {
+        const applicant: applicantType = row.getValue("applicant");
+        const name: string = applicant.name;
+        return <div>{name}</div>;
```
Comment (basokant) [inline src/components/columns.tsx:29]: u can just include name here i think, unnecessary div

Comment (LordExodius) [reply src/components/columns.tsx:29]: 👍

File: src/components/columns.tsx
Comment (LordExodius) [reply src/components/columns.tsx:1]: can have column defs for other tables in this same file

**Review by basokant — APPROVED** [review]

**Pull request merged**